util/fscrypt_completion

```diff
@@ -0,0 +1,225 @@
+# bash completion for fscrypt                              -*- shell-script -*-
```

Comment:
Can you add an Apache license header, like the other source code files in this project?

Reply:
Sure.